lld/MachO/Options.td

```diff
@@ -85,6 +85,9 @@ def icf_eq: Joined<["--"], "icf=">,
     HelpText<"Set level for identical code folding (default: none)">,
     MetaVarName<"[none,safe,all]">,
     Group<grp_lld>;
+def keep_icf_stabs: Joined<["--"], "keep-icf-stabs">,
```

Comment:
`stabs` seems confusing as it appears a single word, but in fact it stands for `symbol table`?
Can we say `keep-icf-symbol` or something? Likewise, I'd use more descriptive word in the following, instead of using `stabs entries` below.

Reply:
The flag name comes from the entries that are created - STAB entries. Not sure of the origin of the name. Some code here: https://github.com/llvm/llvm-project/blob/e3033a2c9494d217d279596fa5b69de8a63a5f87/lld/MachO/SyntheticSections.cpp#L1159 and some docs here: https://wiki.dwarfstd.org/Apple's_%22Lazy%22_DWARF_Scheme.md

It is not referring to the symbol table itself - the symbol table will (I think always) already contain the names of the symbols as they might need to be exported, etc.

Reply:
It appears a known common word for debug info. From your link, I'm seeing this.
> we read the debug map (the stabs entries) out of the executable and create an address translation map for each .o file specifying what address the symbols landed at.

So, I think you can just expand the HelpText a bit on this flag.